.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,29 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.188.0/containers/cpp
+{
+	"name": "C++",
+	"build": {
+		"dockerfile": "Dockerfile",
+		// Update 'VARIANT' to pick an Debian / Ubuntu OS version: debian-10, debian-9, ubuntu-20.04, ubuntu-18.04
```

Comment:
Does changing the variant here changes the Dockerfile? Maybe some extra steps on different distros.

Or should we have one Dockerfile per distro that is different (plus a generic one)?

Reply:
Well i'm not _that_ deep into devcontainers, but I think you'd need a docker-compose file to be able to have multiple devcontainers support (?)

Reply:
I think if "buster" is fine for all debian-based distros for this purpose, I guess it should be fine.

If a problem arises from this later we can change it.

Reply:
Sure.